src/layer/BingTiledImageLayer.js

```diff
@@ -61,14 +61,26 @@ define([
 
             this.attributionImage = WorldWind.configuration.baseUrl + "images/powered-by-bing.png";
 
-            this.attribution = this.createBingLogotype();
+            // Bing logo placement on the screen. Lower right corner by default.
+            this.logoPlacement = new Offset(WorldWind.OFFSET_FRACTION, 1, WorldWind.OFFSET_FRACTION, 0);
```

Comment:
The properties `logoPlacement` and `logoAlignment` need documentation. Otherwise this looks straightforward and well done.